content/posts/September-2020.md

```diff
@@ -22,3 +22,27 @@ draft: false
 [hunor]: https://github.com/csomh
 [matej]: https://github.com/mfocko
 [franta]: https://github.com/lachmanfrantisek
+
+## Week 38 (September 21st - September 18th)
+
+- The combination of the source-git patches with existing patches now works well
+  ([packit#963](https://github.com/packit/packit/pull/963)).
+- Service now mostly support also GitLab as you can see
+  [here](https://gitlab.com/packit-service/hello-world/-/merge_requests/10)
+  or [here](https://gitlab.com/packit-service/src/drpm/-/merge_requests/2).
+  A lot of small issues were fixed in the last week
+  ([packit-service#806](https://github.com/packit/packit-service/pull/806),
+  [packit-service#812](https://github.com/packit/packit-service/pull/812),
+  [packit-service#814](https://github.com/packit/packit-service/pull/814),
+  [packit-service#815](https://github.com/packit/packit-service/pull/815)
+  [packit-service#816](https://github.com/packit/packit-service/pull/816)
+  ).
+- Created Copr projects are now prefixed with the hostname
+  for non-GitHub services and also support multipart namespaces
+  ([packit-service#819](https://github.com/packit/packit-service/pull/819)).
+- [Laura] fixed two issues with the testing-farm, our testing runner service
+  ([packit-service#808](https://github.com/packit/packit-service/pull/808),
+  [packit-service#809](https://github.com/packit/packit-service/pull/809)).
+  It now works also on Pagure and GitLab.
```

Comment:
The two sentences don't feel quite right. The 2 issues belong to the testing-farm or the 'testing runner service' or both? What works on Pagure and Gitlab? The 'testing runner service'?

What about:

- [Laura] fixed two testing-farm related issues ([packit-service#808](https://github.com/packit/packit-service/pull/808), [packit-service#809](https://github.com/packit/packit-service/pull/809)).
- Our testing runner service now works also on Pagure and GitLab.

Reply:
OK, I see it's messy.

- By `our testing runner service` I wanted to describe testing-farm if someone does not know it.
- Those two fixes make the testing works on Pagure and Gitlab.

I'll try to rephrase that.

Reply:
PTAL, thanks